test/unit/math/prim/core/init_threadpool_tbb_test.cpp

```diff
@@ -11,13 +11,12 @@
 
 TEST(intel_tbb_new_init, check_status) {
   set_n_threads(-1);
-  tbb::global_control& tbb_init = stan::math::init_threadpool_tbb();
+  tbb::task_arena& tbb_init = stan::math::init_threadpool_tbb();
   EXPECT_TRUE(tbb_init.is_active());
 
-  EXPECT_EQ(std::thread::hardware_concurrency(),
-            tbb::this_task_arena::max_concurrency());
+  EXPECT_EQ(std::thread::hardware_concurrency(), tbb_init.max_concurrency());
```

Comment:
I'd recommend using `tbb_init.max_concurrency()` over `tbb::this_task_arena::max_concurrency()` because the later returns the total number of cores if the arena isn't initialized.